src/interfaces/sdkTypes.ts

```diff
@@ -55,4 +55,105 @@ export interface LeaderboardUserData {
   totalVolumeInUsdShorts: Decimal;
   totalRealizedPnlPositions: Decimal;
   totalAccruedBorrowingFeesInUsd: Decimal;
-}
\ No newline at end of file
+} 
+export type Market = {
+  id: string;
+  marketName: string;
+  marketSymbol: string;
+  size: string;
+  skew:string;
+  currentFundingRate: string;
+  currentFundingVelocity: string;
+  feedId: string;
+  maxFundingVelocity: string;
+  skewScale: string;
+  makerFee: string;
+  takerFee: string;
+};
+export type Wallet = {
+  id: string;
+  positions?: Position[];
+  orders?: Order[];
+  totalOrdersCount: string;
+  totalPositionsCount: string;
+  openPositionCount: string;
+  countProfitablePositions: string;
+  countLossPositions: string;
+  countLiquidatedPositions: string;
+  totalRealizedPnlPositions: string;
+  totalVolumeInUsd: string;
+  totalVolumeInUsdLongs: string;
+  totalVolumeInUsdShorts: string;
+  totalAccruedBorrowingFeesInUsd?: string;
+};
+
+export type Order = {
+  id: string;
+  market?: Market;
+  user?: Wallet;
+  isLimitOrder: boolean;
+  expectedPrice: string;
+  expectedPriceTime?: string;
+  settlementTime?: string;
+  deadline: string;
+  trackingCode?: string;
+  deltaCollateral: string;
+  deltaSize: string;
```

Comment:
Add field for formatted deltaSize

Reply:
That will be a snapshot of the size in the past based on the price of the token at that timestamp. And here we are adding the formatted delta size in token itself. Both are different